src/controllers/controller.polarArea.js

```diff
@@ -23,6 +22,23 @@ export default class PolarAreaController extends DatasetController {
     };
   }
 
+  parseObjectData(meta, data, start, count) {
+    const {iScale} = meta;
+    const {key = 'key'} = this._parsing;
```

Comment:
I think this "default" will be used when `parsing = true`. Should it be `'r'` or `'value'` rather than `'key'`?
Should probably add a test for that too.

Reply:
`'r'` would sound more logical to me since its the radial scale and woul be more in line with the defaults for the `x` and `y` scale. Although since it behaves more like the doughnut parsing `'value'` might be a better option since its the default there.

@etimberg what do you think about it?

Reply:
I think `r` works. It fits with how we use it for the `radar` chart where it means the distance out from the center point.